### ISSUE TITLE
code editor is crashing in blender 3.6

### ISSUE THREAD

**Issue by Andrej730** [K-410/blender-scripts#15]
How to reproduce:
1) Open text editor
2) create new text file
3) try to type

Traceback:
```
Error: Python: Traceback (most recent call last):
  File "\Blender\3.6\scripts\addons\code_editor.py", line 925, in invoke
    return bpy.ops.ce.scroll('INVOKE_DEFAULT')
  File "C:\software\Blender_Launcher\daily\blender-3.6.0-beta+daily.58b5d38824de\3.6\scripts\modules\bpy\ops.py", line 111, in __call__
    ret = _op_call(self.idname_py(), C_dict, kw, C_exec, C_undo)
RuntimeError: Error: Python: Traceback (most recent call last):
  File "\Blender\3.6\scripts\addons\code_editor.py", line 889, in invoke
    return self.scroll_timer(context, event)
  File "\Blender\3.6\scripts\addons\code_editor.py", line 899, in scroll_timer
    nlines = round(0.3 * (center + self.ce.slide - mry) / mlh)
AttributeError: 'CodeEditorMain' object has no attribute 'slide'
Location: C:\software\Blender_Launcher\daily\blender-3.6.0-beta+daily.58b5d38824de\3.6\scripts\modules\bpy\ops.py:111
```

**Comment by K-410** [comment]
Hi,

just tested on 3.6.0 RC build a07063e6a403 (june 23) and I'm not seeing any issues.

That error suggests that the addon isn't initialized properly. I would check the console for additional errors.

Can you test on the latest 3.6.0?

**Comment by Andrej730** [comment]
updated to the latest version and it's now fixed, just as #16
